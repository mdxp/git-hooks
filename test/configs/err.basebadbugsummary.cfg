[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	ignorecase = true
[gate]
	name = gate
	notify = invalidemailaddress@abadaddress
	user-check = skip
	comment-check = skip
	review-check = skip
[gate "the_epa"]
	comment-check = yes
