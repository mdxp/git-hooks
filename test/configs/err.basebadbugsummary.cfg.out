remote: commit lines must be of the form <id> <summary>[K
remote: > new file[K
remote: 
remote: Use the following command to edit the last commit message:[K
remote:   git commit --amend --reset-author[K
remote: 
